File: random_name_generator.R
######################################
# Create a random ID Name of length 12
# @author: pm #
######################################


mkRandomString <- function(len=12){
    n = 1;
    randomString <- c(1:n)  # initialize vector
    for (i in 1:n){
        randomString[i] <- paste(sample(c(0:9, letters, LETTERS),
                                        len, replace=TRUE),
                                 collapse="") ;
    }
    return(randomString)
}

print(mkRandomString(12))

File: random_name_generator.py
# -*- coding: utf-8 -*-
"""
Create a random ID Name of length 12
@author: pm
"""

import string, random

def id_generator(size=12, chars=string.ascii_uppercase + string.ascii_lowercase + string.digits):
    return ''.join(random.choice(chars) for _ in range(size))

print id_generator(12)